dd-java-agent/instrumentation/jdbc/src/main/java/datadog/trace/instrumentation/jdbc/StatementInstrumentation.java

```
@@ -70,7 +73,7 @@ public void methodAdvice(MethodTransformer transformer) {
   }
 
   public static class StatementAdvice {
-    @Advice.OnMethodEnter(suppress = Throwable.class)
```

why the suppression has been removed?

---

ok found the answer below. :)

---

That was in purpose to let blocking exception bubble-up from `StatementAdvice` to http server advices.
The idea is - to suppress everything except `BlockingException`. So, instead of `suppress = Throwable.class`, I used custom `try-catch` to suppress exceptions.